packages/elements-core/src/components/TryIt/Parameters/parameter-utils.ts

```diff
@@ -68,6 +68,14 @@ export function getPlaceholderForParameter(parameter: ParameterSpec) {
   return String(parameter.schema?.type ?? '');
 }
 
+export function getPlaceholderForSelectedParameter(parameter: ParameterSpec) {
+  const { value: parameterValue, isDefault } = getValueForParameter(parameter);
+
+  if (parameterValue && isDefault) return `defaults to: ${parameterValue}`;
```

Comment:
what if the value is `0` or empty string?

Reply:
fair point